privacyidea/lib/tokens/pushtoken.py

```diff
@@ -309,6 +323,7 @@ def get_init_detail(self, params=None, user=None):
         user = user or User()
         tokenlabel = params.get("tokenlabel", "<s>")
         tokenissuer = params.get("tokenissuer", "privacyIDEA")
+        sslverify = params.get(PUSH_ACTION.SSL_VERIFY, "1")
```

Comment:
I think it would be safer to add a check that ``sslverify`` is either "1" or "0". Otherwise, we add whatever is passed as ``push_ssl_verify`` to the query string below :)

Reply:
OK

Reply:
done in f1ae0319